<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Step 1: Enter username]]>
    </line>
  </reporter-output>
  <suite started-at="2025-08-18T07:46:13 IST" name="Default suite" finished-at="2025-08-18T07:46:20 IST" duration-ms="7034">
    <groups>
    </groups>
    <test started-at="2025-08-18T07:46:13 IST" name="Default test" finished-at="2025-08-18T07:46:20 IST" duration-ms="7034">
      <class name="org.wipro.auto.uo.testcases.LoginScenario_log">
        <test-method is-config="true" signature="setupReport()[pri:0, instance:org.wipro.auto.uo.testcases.LoginScenario_log@5dafbe45]" started-at="2025-08-18T07:46:13 IST" name="setupReport" finished-at="2025-08-18T07:46:14 IST" duration-ms="693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:org.wipro.auto.uo.testcases.LoginScenario_log@5dafbe45]" started-at="2025-08-18T07:46:14 IST" name="launchBrowser" finished-at="2025-08-18T07:46:18 IST" duration-ms="4432" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="tc01_validate_valid_login_func()[pri:0, instance:org.wipro.auto.uo.testcases.LoginScenario_log@5dafbe45]" started-at="2025-08-18T07:46:18 IST" name="tc01_validate_valid_login_func" finished-at="2025-08-18T07:46:19 IST" duration-ms="1160" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Product] but found [Products]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Product] but found [Products]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at org.wipro.auto.uo.testcases.LoginScenario_log.tc01_validate_valid_login_func(LoginScenario_log.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Step 1: Enter username]]>
            </line>
          </reporter-output>
        </test-method> <!-- tc01_validate_valid_login_func -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:org.wipro.auto.uo.testcases.LoginScenario_log@5dafbe45]" started-at="2025-08-18T07:46:19 IST" name="closeBrowser" finished-at="2025-08-18T07:46:19 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="flushReport()[pri:0, instance:org.wipro.auto.uo.testcases.LoginScenario_log@5dafbe45]" started-at="2025-08-18T07:46:19 IST" name="flushReport" finished-at="2025-08-18T07:46:20 IST" duration-ms="657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushReport -->
      </class> <!-- org.wipro.auto.uo.testcases.LoginScenario_log -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
